src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -236,6 +244,13 @@ def __init__(self, file_handle):
         """Initialize the object."""
         self._file_handle = file_handle
 
+    def get_mesh_type(self) -> MeshType:
+        """Returns the type of the mesh."""
+        if "cells" in self._file_handle["meshes"]["1"].keys():
```

Comment:
I wonder if we should try/except this and return `MeshType.UNKNOWN` in case of an exception.

Reply:
Done, Thank you.